src/__tests__/Dropdown.spec.js

```diff
@@ -1,388 +1,320 @@
+
+    rerender(
+      <Dropdown isOpen toggle={toggle} data-testid="drpdwn">
+        <DropdownToggle>Toggle</DropdownToggle>
+        <DropdownMenu>
+          <DropdownItem>First item</DropdownItem>
+          <DropdownItem>Second item</DropdownItem>
+          <DropdownItem id="divider" divider />
+        </DropdownMenu>
+      </Dropdown>,
+    );
+
+    return { rerender };
+  }
+
+  it('should render a single child', () => {
+    render(<Dropdown isOpen>Ello world</Dropdown>);
+
+    expect(screen.getByText(/ello world/i)).toHaveClass('dropdown');
   });
 
-  it('should render multiple children when isOpen is true', () => {
-    isOpen = true;
-    const wrapper = mount(
-      <Dropdown isOpen={isOpen} toggle={toggle}>
+  it('should render menu when isOpen is true', () => {
+    render(
+      <Dropdown isOpen>
         <DropdownToggle>Toggle</DropdownToggle>
         <DropdownMenu>
           <DropdownItem>Test</DropdownItem>
         </DropdownMenu>
       </Dropdown>,
     );
 
-    expect(wrapper.find('.btn').hostNodes().text()).toBe('Toggle');
-    expect(wrapper.find('.dropdown').hostNodes().length).toBe(1);
-    expect(wrapper.find('.dropdown-item').hostNodes().length).toBe(1);
-    expect(wrapper.childAt(0).childAt(0).children().length).toBe(2);
+    expect(screen.getByText(/toggle/i)).toHaveClass('btn');
+    expect(screen.getByText(/test/i)).toHaveClass('dropdown-item');
   });
 
   it('should not call props.toggle when disabled ', () => {
-    isOpen = true;
-    let props = createSpyObj('props', ['toggle']);
-    const wrapper = mount(
-      <Dropdown isOpen={isOpen} toggle={props.toggle} disabled>
+    const toggle = jest.fn();
+    render(
+      <Dropdown isOpen toggle={toggle} disabled>
         <DropdownToggle>Toggle</DropdownToggle>
         <DropdownMenu>
           <DropdownItem>Test</DropdownItem>
         </DropdownMenu>
       </Dropdown>,
     );
-    const instance = wrapper.instance();
 
-    instance.toggle({ preventDefault: () => {} });
-    expect(props.toggle).not.toHaveBeenCalled();
+    user.click(screen.getByText(/toggle/i));
+    expect(toggle).not.toHaveBeenCalled();
   });
 
   describe('handleProps', () => {
     it('should not pass custom props to html attrs', () => {
-      const wrapper = mount(
-        <Dropdown inNavbar a11y isOpen={isOpen} toggle={toggle}>
+      const toggle = jest.fn();
+      render(
+        <Dropdown a11y isOpen toggle={toggle}>
           <DropdownToggle>Toggle</DropdownToggle>
           <DropdownMenu>
             <DropdownItem>Test</DropdownItem>
           </DropdownMenu>
         </Dropdown>,
       );
 
-      expect(wrapper.find('.dropdown').prop('inNavbar')).toBe(undefined);
-      expect(wrapper.find('.dropdown').prop('toggle')).toBe(undefined);
-      expect(wrapper.find('.dropdown').prop('a11y')).toBe(undefined);
-      expect(wrapper.find('.dropdown').prop('isOpen')).toBe(undefined);
+      const dropdown = document.getElementsByClassName('dropdown')[0];
```

Comment:
Not 100%, but you might be able to get this element from `const { container } = render(...)` where container is dropdown

Reply:
I think `container` returns a `div` above the `Dropdown` and not dropdown itself. And it is a fiber node and not a DOM node. I'm also not sure tbh, but was unable to make it work with `container`

Something like 
```
<body>
  <div> // container is this
    <Dropdown />
  </div>
<body>
```

Reply:
I see, thanks for checking that out!